esphome/components/atm90e32/button/atm90e32_button.cpp

```diff
@@ -1,20 +1,78 @@
 #include "atm90e32_button.h"
+#include "esphome/core/component.h"
 #include "esphome/core/log.h"
 
 namespace esphome {
 namespace atm90e32 {
 
-static const char *const TAG = "atm90e32.button";
+static const char *const TAG = "CALIBRATION Button";
```

Comment:
This should not change
```suggestion
static const char *const TAG = "atm90e32.button";
```